SEED-platform/seed
# build: use alpine 3.14 base image

#### Any background context you want to provide?
SEED's docker images were building off of an older SEED image which itself was built on an EOL'd alpine image.
This is the first step in the larger effort of updating SEED's dependencies

#### What's this PR do?
- Uses alpine 3.14 as base image for Dockerfile and Dockerfile-dev. As a result, this also:
  - Updates python to 3.9
  - Updates pint to v0.17 (owned by django-pint)
  - Possibly updates other dependencies but I'm not sure what they are
- Updates SEED's openlayers (ol)
- Fixes a bug in the data auto-population feature on homepage (payload to column profile was missing a key)

#### How should this be manually tested?
As mentioned in the updated docs, if you're on OSX you need to update docker desktop to version 3.0 or later. Just redownload docker desktop and replace the app in your applications dir.

You should be able to rebuild the web/celery image and start docker-compose and view the site. Extensive testing will be implicit over this quarter as we work on other things.
```bash
# this might take a while (1+ hour) due to our python dependencies
docker-compose -f docker-compose.yml -f docker-compose.dev.yml build

docker-compose -f docker-compose.yml -f docker-compose.dev.yml up
```

#### What are the relevant tickets?
#2433 

#### Screenshots (if appropriate)
